commit 34c390a09635dd318c917bc89c182d8c2b538cfc
Author: Noah Watkins <nwatkins@redhat.com>
Date:   Mon Jul 30 15:11:05 2018 -0700

    Revert "mgr/pybind: expose erasure code profiles"
    
    The erasure code profiles are exposed through the OSD dump which was
    overlooked on inspection.
    
    This reverts commit b5487abb8b946c534d0e93f88c8b9b2d1cb35bae.
    
    Signed-off-by: Noah Watkins <nwatkins@redhat.com>

diff --git a/src/mgr/ActivePyModules.cc b/src/mgr/ActivePyModules.cc
index 8ffa7fe2ea6..6da3576bbb3 100644
--- a/src/mgr/ActivePyModules.cc
+++ b/src/mgr/ActivePyModules.cc
@@ -362,13 +362,6 @@ PyObject *ActivePyModules::get_python(const std::string &what)
       mgr_map.dump(&f);
     });
     return f.get();
-  } else if (what == "ec_profiles") {
-    PyFormatter f;
-    cluster_state.with_osdmap([&f, &what](const OSDMap &osdmap){
-      const auto &profiles = osdmap.get_erasure_code_profiles();
-      osdmap.dump_erasure_code_profiles(profiles, &f);
-    });
-    return f.get();
   } else {
     derr << "Python module requested unknown data '" << what << "'" << dendl;
     Py_RETURN_NONE;
